### DISCUSSION TITLE
可以通过内网穿透来实现局域网联机或者加速P2P联机吗

### DISCUSSION THREAD

**Discussion by the-only-noob** [natfrp/wiki#123] [使用问题]
可以通过内网穿透来实现局域网联机或者加速P2P联机吗，比如文明6，桌游模拟器之类的
文明6支持P2P和局域网联机，但是P2P联机总是掉，所以我想通过内网穿透让好友访问局域网来实现加速
(当然能P2P加速就更好了)

**Comment by Klosw** [comment]
内网穿透可以实现局域网联机 

 p2p 我就不知道怎么处理了

**Comment by Klosw** [comment]
你只要设置好穿透的 端口  然后你朋友访问 你网页上的 地址和端口  就能实现 局域网加速了
